demos/mri_reconstruction_demo/cpp/main.cpp

```diff
@@ -0,0 +1,167 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <inference_engine.hpp>
+#include <opencv2/opencv.hpp>
+
+#include "mri_reconstruction_demo.hpp"
+#include "npy_reader.hpp"
+
+using namespace InferenceEngine;
+
+bool ParseAndCheckCommandLine(int argc, char *argv[]);
+
+static cv::Mat infEngineBlobToMat(const Blob::Ptr& blob);
+
+struct MRIData {
+    cv::Mat data;
+    cv::Mat samplingMask;
+    cv::Mat reconstructed;
+    // Hybrid-CS-Model-MRI/Data/stats_fs_unet_norm_20.npy
+    std::vector<double> stats = {2.20295299e-1, 1.11048916e+3, 4.16997984, 4.71741395};
```

Comment:
```suggestion
    static constexpr double stats[]{2.20295299e-1, 1.11048916e+3};
```
For python you probably don't need the last two values as well

Reply:
Why vector? Array fits here. Or you can give these values dedicated descriptive values like scale, mean